static/src/javascripts/projects/commercial/modules/hosted/onward-journey-carousel.js

```diff
@@ -0,0 +1,80 @@
+// @flow
+import qwery from 'qwery';
+import fastdom from 'fastdom';
+import $ from 'lib/$';
+
+class HostedCarousel {
+    $carousel: Object;
+    $nextItem: Object;
+    $prevItem: Object;
+    $dots: Object;
+    pageCount: number;
+    index: number;
+
+    moveCarouselBy(direction: number) {
+        this.moveCarouselTo(this.index + direction);
+    }
+
+    moveCarouselTo(index: number) {
+        const that = this;
```

Comment:
This (= `that`) should not be necessary anymore, since you are using the fat-arrows further down.